std/string.d

```diff
@@ -457,26 +457,27 @@ pure nothrow @system unittest // https://issues.dlang.org/show_bug.cgi?id=15136
 alias CaseSensitive = Flag!"caseSensitive";
 
 /++
-    Searches for character in range.
+    Searches for character `c` in string or range `s` starting at index
+    `startIdx`. If `startIdx` is not given, it defaults to 0.
```

Comment:
No need to describe each individual parameter in the [summary][1]. This part of the documentation is meant to give a brief, non-exhaustive overview that can be quickly scanned (like the first line of a good git commit message). Details like this belong in either the [description][2] or the `Params:` section.

Suggested change:
```suggestion
    Searches for a character in a string or range.
```

[1]: https://dlang.org/spec/ddoc.html#summary
[2]: https://dlang.org/spec/ddoc.html#description